### PR TITLE
Creates unit test for trials call

### PR DESCRIPTION
**Pull request by reynoldtan**

### CODE REVIEW

File: calls/v1/trials/TripalWebServiceBrapiV1Trials.inc [hunk 1]
```diff
@@ -233,7 +235,7 @@ class TripalWebServiceBrapiV1Trials extends TripalWebServiceDatabaseCall {
           }
         }
 
-        $project->studies = [ $all_studies ];
+        $project->studies = (array) ['set' => $all_studies];
```
Comment (laceysanderson) [inline calls/v1/trials/TripalWebServiceBrapiV1Trials.inc:238]: This line makes no sense... Casting an associative array as an array will accomplish nothing... instead $all_studies should be an array of objects and used directly.

**Review by laceysanderson — CHANGES_REQUESTED** [review]
Code Changes ❌ 
 - Studies are not being added to the trial properly

Check in the Browser:
 - Structure matches Apiary 
     - active is the wrong type. should be true ❌ 
 - Returns results using test data ✅
 - Check parameters
     - commonCropName ✅
     - programDbId ✅
     - missing parameters were not tested ❌ 
 
Test Review:
 - Tests all parameters ❌ 
     - Missing locationDbId, active, sortBy, sortOrder
 - Includes correct data structure and data types based on Apiary
     - Active should be a boolean ❌ 

There should be example data for studies within a trial ❌ 

Tests Pass ✅

**Comment by reynoldtan** [comment]
There should be example data for studies within a trial
Please try host/web-services/brapi/v1/trials?pageSize=10&page=3

**Comment by laceysanderson** [comment]
Great. This shows that your implementation is not the correct format.
Screen Shot 2021-02-01 at 12 09 42 PM
Screen Shot 2021-02-01 at 12 09 14 PM

As I mentioned in my comment 'studies' should be an array of objects. Thus, $all_studies should be an array of objects and used directly. There is also incorrect names (i.e. `location` ), capitalization (i.e. `studydbid`), and fields (i.e. `phenotype_id`).

**Comment by reynoldtan** [comment]
I understand the structure of studies, this is the result from my end. Could you please clear your cache and give it a quick refresh? 
Screen Shot 2021-02-01 at 12 50 18 PM
.

**Pull request merged**